experimental/CHANGELOG.md

```diff
@@ -15,6 +15,25 @@ All notable changes to experimental packages in this project will be documented
     * `createInstrumentDescriptor`
     * `createInstrumentDescriptorWithView`
     * `isDescriptorCompatibleWith`
+* chore(api-metrics): clean up exports [#3198](https://github.com/open-telemetry/opentelemetry-js/pull/3198) @pichlermarc
+  * removes export for:
+    * `NoopMeter`
+    * `NoopMetric`
+    * `NoopCounterMetric`
+    * `NoopUpDownCounterMetric`
+    * `NoopHistogramMetric`
+    * `NoopObservableMetric`
+    * `NoopObservableCounterMetric`
+    * `NoopObservableGaugeMetric`
+    * `NoopObservableUpDownCounterMetric`
+    * `NOOP_COUNTER_METRIC`
+    * `NOOP_HISTOGRAM_METRIC`
+    * `NOOP_UP_DOWN_COUNTER_METRIC`
+    * `NOOP_OBSERVABLE_COUNTER_METRIC`
+    * `NOOP_OBSERVABLE_GAUGE_METRIC`
+    * `NOOP_OBSERVABLE_UP_DOWN_COUNTER_METRIC`
+    * `NoopMeterProvider`
```

Comment:
Nit: would you mind sorting the list? i.e. move this item above the NOOP instances.

Reply:
Oh just saw that @dyladan did that in [5581d5b](https://github.com/open-telemetry/opentelemetry-js/pull/3198/commits/5581d5b302af309094b529ee4acf104ce79784b7). Thanks for the feedback (and thx to Dan for sorting) :slightly_smiling_face: